mantidproject/mantid
# Moved remaining new release notes and removed unused headings

**Description of work.**

Moved the remaining new release notes from the release period into the main release note files.
Removed all the unused headings from the main release note files.

**To test:**

 - Build `docs-html`
 - Open all the main release note html pages (including `index.html`)
 - Check new release notes to have good spelling and grammar
 - Check that any links go to where is expected
 - Check that there are no unused headings
 - Final look over each file

*There is no associated issue.*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
